src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -49,8 +48,8 @@ type UserConfig interface {
 
 // NewFetcher creates a new configuration fetcher with the specified
 // interface for obtaining the user ID and token if those are populated.
-func NewFetcher(conf UserConfig, httpFetcher util.HTTPFetcher) Fetcher {
-	return &fetcher{lastCloudConfigETag: map[string]string{}, user: conf, httpFetcher: httpFetcher}
+func NewFetcher(conf UserConfig, httpClient *http.Client) Fetcher {
```

Comment:
As mentioned in another comment, our autoupdate relies on actually having an http.Client.  It doesn't have to, and I could go in and refactor that too if you want.  Regarding the interface, I don't see why we should redefine what's already defined in `http.RoundTripper`.